src/app/utility/predictor-data/predictor-data.component.html

```diff
@@ -5,68 +5,49 @@ <h4>Predictors</h4>
         consequat.</p>
 
     <div class="controls mb-2">
+        <button class="btn btn-outline" (click)="editPredictors()">Edit Predictors</button>
```

Comment:
I don't love the language and buttons but this doesn't effect your intended functionality